### PR TITLE
fix: Use import to avoid circular import in generate_demo_data

### PR DESCRIPTION
**Pull request by tomasfarias**
## Problem

The `demo_create_data` submodule is loaded upon initializing the `tasks` module, as `tasks` uses a `from . import demo_create_data` statement.

This causes the `demo_create_data` submodule to also run its import statements, including attempting to `from posthog.demo.matrix.manager import MatrixManager`.

This produces an import cycle when running `generate_demo_data` as the `manager` submodule (required by the `generate_demo_data` command) imports a task from the `tasks` module.

## Changes

Using `import posthog.demo.matrix.manager as manager` instead of `from posthog.demo.matrix.manager import MatrixManager` avoids the cycle, at the cost of an alias. 

This is probably the simplest solution, but a more comprehensive one would have refactored the initialization of the `tasks` module to use `import ...` instead of `from . import ...`. However that would have required more understanding of where are the submodules exposed by `tasks` used, and higher risk overall.

## How did you test this code?

Follow the [Develop Locally documentation](https://posthog.com/handbook/engineering/developing-locally) until you get to step 6 and run the recommended command to generate test data:

```sh
DEBUG=1 ./manage.py generate_demo_data -h
```

Or just run the command if you have already setup PostHog locally.

`master` branch fails with a:

```sh
File "/home/tomasfarias/src/github.com/PostHog/posthog/posthog/tasks/demo_create_data.py", line 4, in <module>
    from posthog.demo.matrix.manager import MatrixManager
ImportError: cannot import name 'MatrixManager' from partially initialized module 'posthog.demo.matrix.manager' (most likely due to a circular import) (/home/tomasfarias/src/github.com/PostHog/posthog/posthog/demo/matrix/manager.py)
```

Where as patch works as expected.

A unit test that uses `call_command` could be added, however when running the full testing pipeline the import error is missed. Only when the test is running alone does it pop up. This is likely due to `MatrixManager` being imported by a test running before, so Python can avoid the cycle by using this instead. Perhaps looking into deleting the module from `sys.modules` could work...

### CODE REVIEW

File: posthog/tasks/demo_create_data.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from celery import shared_task
 from sentry_sdk import capture_exception
 
-from posthog.demo.matrix.manager import MatrixManager
+import posthog.demo.matrix.manager as manager
```
Comment (Twixes) [inline posthog/tasks/demo_create_data.py:4]:
```suggestion
from posthog.demo.matrix import manager```

Comment (raquelmsmith) [reply posthog/tasks/demo_create_data.py:4]: Just a heads up, I don't think you actually want those three backticks there. ` ``` `

**Review by Twixes — COMMENTED** [review]
It'd be great to have a test for the `generate_demo_data` command, but when I tried adding one just before Christmas, it was mysteriously getting stuck, so I dropped it 😅

**Comment by raquelmsmith** [comment]
Thank you for this!! It was on my list to fix since it occurred when we moved the demo data creation to a task. I think we will actually make the task synchronous again, but let's merge this since I'm not 100% certain when that will happen.

**Pull request merged**